salt/elasticsearch/plugins/readonlyrest.sls

```diff
@@ -39,6 +39,27 @@ readonlyrest-install:
       - pkg: /opt/pem-to-keystore.sh
       - file: /opt/pkcs-password
 
+/opt/pem-to-keystore-wrapper.sh:
+  file.managed:
+    - name: /opt/pem-to-keystore-wrapper.sh
+    - source: salt://elasticsearch/files/pem-to-keystore-wrapper.sh
+    - mode: 755
+    - user: root
+    - group: root
+    - require:
+      - file: /opt/pem-to-keystore.sh
+
+/etc/sudoers.d/90-pem-to-keystore:
+  file.managed:
+    - user: root
+    - group: root
+    - mode: 440
+    - source: salt://elasticsearch/files/sudoers.d/pem-to-keystore
+    # check_cmd is appended with a temporary file containing the new managed contents.
+    - check_cmd: visudo -c -f
```

Comment:
Doesn't `-f` take an argument?

Reply:
Yes `-f` does indeed require a file path - this is why I put the comment above.
Salt appends the path for you, it is nice because salt can use the temporary upload location, testing it before it is implemented.

Reply:
Aha, I'll make a suggestion to clarify that.